index.html

```diff
@@ -9,6 +9,17 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
+    <div class="popup-links untoggled">
+      <button type="button" class="close-popup"> <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M23.954 21.03l-9.184-9.095 9.092-9.174-2.832-2.807-9.09 9.179-9.176-9.088-2.81 2.81 9.186 9.105-9.095 9.184 2.81 2.81 9.112-9.192 9.18 9.1z"/></svg> </button>
+      <ul>
+        <li class="bigger"><a class="nav-link bigger">Portoflio</a></li>
+        <li class="bigger"><a class="nav-link bigger">About</a></li>
+        <li class="bigger">
+          <a class="nav-link bigger">
+            <i>Contact</i></a>
+        </li>
+      </ul>
```

Comment:
You did a really great job but kindly fix the routes of nav links to the specific sections for example when you click the link "About" to send you to the about me section. Keep coding! 💪